README.md

```diff
@@ -387,6 +387,7 @@ The USB Interface Chip is the microcontroller placed close to the battery connec
 
 All these projects contain steps and resources required for reproduction.
 
+- [Mewt](https://github.com/dakota-mewt/mewt/blob/main/README_Microbit.md) - Mute/unmute all computer microphones, displays status and hot mic using volume meter
```

Comment:
Project looks really cool, thanks for the submission!
Let's move it to the end of the list and we can merge it 👍